src/app/pages/fund/components/fiat-provider-item.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, color, Stack } from '@stacks/ui';
+
+import { Caption, Title } from '@app/components/typography';
+import { FundPageSelectors } from '@tests/page-objects/fund.selectors';
+import { FastCheckoutBadge } from './fast-checkout-badge';
+import { ZeroPercentFeesBadge } from './zero-percent-fees-badge';
+
+interface FiatProviderProps {
+  icon: string;
+  onGoToProvider(): void;
+  showFastCheckoutBadge: boolean;
+  showZeroPercentFeesBadge: boolean;
+  title: string;
+}
+export const FiatProviderItem = (props: FiatProviderProps) => {
+  const { icon, onGoToProvider, showFastCheckoutBadge, showZeroPercentFeesBadge, title } = props;
+
+  return (
+    <Box
+      _focus={{
+        boxShadow: ['0px 8px 16px rgba(0, 0, 0, 0.08)', '0 0 0 3px #5546FF'],
+      }}
+      _hover={{
+        boxShadow: '0px 8px 16px rgba(0, 0, 0, 0.08)',
+        cursor: 'pointer',
+      }}
+      as="button"
+      border="1px solid"
+      borderColor={color('border')}
+      borderRadius="16px"
+      boxShadow="0px 1px 2px rgba(0, 0, 0, 0.04)"
+      data-testid={FundPageSelectors[`Btn${title}` as keyof typeof FundPageSelectors]}
+      display="flex"
+      onClick={onGoToProvider}
+      textAlign="left"
+    >
+      <Stack alignItems="flex-start" maxWidth={['unset', '280px']} p="loose" spacing="base">
+        <Stack alignItems="center" isInline spacing="base">
+          <Box
+            alignItems="center"
+            border="1px solid"
+            borderColor="#DCDDE2"
+            borderRadius="10px"
+            display="inline-flex"
+            height="40px"
+            justifyContent="center"
+            width="40px"
+          >
+            <img src={icon} width="24px" />
+          </Box>
+          <Title>{title}</Title>
+        </Stack>
```

Comment:
You can never overuse Stack 😆  ...hard to say but happy to change.

Reply:
- Maybe we should for `grid` instead, since this is definitely a gridded layout. I believe it will be much easier to control and make responsive. 
- I'm tempted to go for a more fluid responsive behavior, because at some viewport sizes (in between breakpoints) the cards sometimes look odd. I can look for reference or toy with the CSS to show you what I mean.
- In larger viewports (when more than 1 card fits should enforce a left/right margin of 48px for the cards container
- Content should not be centered, to prevent single card rows to fit within the grid.

Reply:
This is the issue with centering (when the cards are uneven)
![CleanShot 2022-05-24 at 16 35 18@2x](https://user-images.githubusercontent.com/239215/170062008-ea015ea3-7cb7-49d8-80d7-edfe058a6e03.png)

Reply:
> Isn't this what I'm doing when I use columnGap and rowGap applied to the Flex component? I can change to use our Grid component if want?

Yes, technically it's similar since `grid` didn't exist when `flexbox` was designed. You can use grid with `grid-gap` and `column-gap` to achieve the same thing. https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
2 x 2 will look best